trailblazer/apps/slurm/models.py

```diff
@@ -58,19 +59,16 @@ class SqueueResult(BaseModel):
     """Model used to parse SLURM squeue output."""
 
     jobs: List[SqueueJob]
-    jobs_status_distribution: Optional[Dict[str, float]]
+    jobs_status_distribution: Optional[Dict[str, float]] = None
 
-    @validator("jobs_status_distribution", always=True)
-    def set_jobs_status_distribution(
-        cls, _: Optional[List[SqueueJob]], values: Dict[str, Any]
-    ) -> Optional[Dict[str, float]]:
+    @model_validator(mode="after")
```

Comment:
Tricky to know which one should be used (before, after or wrap). To me it seems reasonable to always use after? 
https://docs.pydantic.dev/latest/usage/validators/#before-after-wrap-and-plain-validators